php/apache2/install.sls

```diff
@@ -1,18 +1,23 @@
-{% from "php/map.jinja" import php with context %}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import php with context %}
+{%- from tplroot ~ "/libtofs.jinja" import files_switch with context %}
 
-{% set state = 'apache2' %}
-{% include "php/installed.jinja" %}
+{%- set state = 'apache2' %}
+{%- include tplroot ~ "/installed.jinja" %}
 
-{% if grains['os_family'] == "FreeBSD" %}
+{%- if grains['os_family'] == "FreeBSD" %}
 {{ php.lookup.apache2.module_config }}:
   file.managed:
-    - source: salt://php/apache2/files/mod_php.conf.jinja
+    - source: {{ files_switch( ['mod_php.conf.jinja'],
+                               'php_apache2_module_config',
+                               v1_path_prefix = '/apache2'
```

Comment:
```suggestion
```

* `v1_path_prefix` has been removed from the latest `libtofs.jinja`.

Reply:
Used `use_subpath`